bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties

```diff
@@ -40,6 +42,29 @@ Dialog_UseTourTypeMappingModified_Title = Tour Type Mapping Usage Modified
 
 Dialog_ValidatingSuuntoTokens_SubTask = \u231B Validating the Suunto App tokens
 
+Filename_Component_ActivityId        = Activity Id
+Filename_Component_ActivityId_Abbr   = ID
+Filename_Component_ActivityType      = Activity Type
+Filename_Component_ActivityType_Abbr = TYPE
+Filename_Component_Day               = Day\u0009
+Filename_Component_Day_Abbr          = DD
+Filename_Component_FitExtension      = Fit Extension
+Filename_Component_FitExtension_Abbr = Fit
+Filename_Component_Hour              = Hour
+Filename_Component_Hour_Abbr         = HH
+Filename_Component_Minute            = Minute
+Filename_Component_Minute_Abbr       = MM
+Filename_Component_Month             = Month
+Filename_Component_Month_Abbr        = MM
```

Comment:
Month (MM) and minute (MM) had the same abbreviation -> fixed in https://github.com/mytourbook/mytourbook/commit/7f6a84ccae1c85a9decd4d5edbf0cd4a97fc410f by using (mm) for minutes and (hh) for hours

Reply:
I fixed it for the french translation too